### ISSUE TITLE
rule-no-duplicate-properties font-face declaration

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#533]
I get error this code:
```css
@font-face {
    font-family: "Glyphicons Halflings";
    src: url("../fonts/glyphicons-halflings-regular.eot");
    src:
        url("../fonts/glyphicons-halflings-regular.eot?#iefix") format("embedded-opentype"),
        url("../fonts/glyphicons-halflings-regular.woff2") format("woff2"),
        url("../fonts/glyphicons-halflings-regular.woff") format("woff"),
        url("../fonts/glyphicons-halflings-regular.ttf") format("truetype"),
        url("../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular") format("svg");
}
```
But it is normal add font-face to your css. Maybe add options to ignore some properties?

**Comment by jeddy3** [comment]
> Maybe add options to ignore some properties?

Sgtm. 

@davidtheclark We've discussed optional options for `*-no-*` rules before, but I don't think we ended up implementing any. This seems like a sensible candidate for the first one e.g.

```json
"rule-no-duplicate-properties": [ true, { "ignore": [ "src" ] } ]
```

What do you think?

**Comment by evilebottnawi** [comment]
@jeddy3 if need pull request with test, let me know :)

**Comment by jeddy3** [comment]
@evilebottnawi That would be awesome. Thanks for offering to help. :)

Best wait for David to feedback on the config signature before making a start on it though :)

**Comment by davidtheclark** [comment]
Config signature looks good -- but I'm wondering if we should always ignore `src`. Does it only occur in `font-face`? If so, since in `font-face` it's very common to repeat that property, I'd say we should always ignore it instead of making the user hit the problem, look up the solution, etc.

**Comment by jeddy3** [comment]
> Does it only occur in font-face? 

I think so. Struggling to find much about the `src` property, other than in the [font-face spec](http://www.w3.org/TR/css3-fonts/#src-desc).

> I'd say we should always ignore it instead of making the user hit the problem, look up the solution, etc.

So, this sgtm :)

**Comment by davidtheclark** [comment]
Closed by #542.
